magicctapipe/scripts/lst1_magic/lst1_magic_dl2_to_dl3.py

```diff
@@ -117,8 +118,39 @@ def dl2_to_dl3(input_file_dl2, input_dir_irf, output_dir, config):
     else:
         pnt_az_mean = pnt_az_wrap_180deg.mean().wrap_at("360 deg").to_value("rad")
 
-    target_point = np.array([pnt_coszd_mean, pnt_az_mean])
-    logger.info(f"\nTarget point in (cos(Zd), Az): {target_point.round(5).tolist()}")
+    distances = np.degrees(
+        angular_separation(
+            pnt_az_mean,
+            np.pi / 2 - np.arccos(pnt_coszd_mean),
+            irf_data["grid_points"][:, 1],
+            np.pi / 2 - np.arccos(irf_data["grid_points"][:, 0]),
+        )
+    )
```

Comment:
Here I would use `Angle` e.g. 

```python
distances = Angle(
        angular_separation(
            pnt_az_mean,
            np.pi / 2 - np.arccos(pnt_coszd_mean),
            irf_data["grid_points"][:, 1],
            np.pi / 2 - np.arccos(irf_data["grid_points"][:, 0]),
        ), unit=u.rad
    ).to(u.degree)
```

and then later

```python
max_distance = config_dl3.pop("max_distance") * u.degree
```

so that it is clear that `max_distance` is in degrees (and `distances` as well)

Reply:
I modified the code to use Quantity for angles